### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by andym80**
This project was made from a mobile version parsed from a Figma template and converted to a desktop version. 

All files CSS files was linked with media-queries on file index.html & stylesbg.css. (desktop) styles.css (mobile)

All linters was installed  .hintrc .stylelintrc  linters.yml

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,14 +72,15 @@ <h3 class="connectlink">LET´S CONNECT</h3>
           </div>
         </div>
       </div>
+    </div>
     </section>
 
     <section class="works">
```
Comment (Kossi-stack) [inline index.html:80]: - [ ] Kindly fix the work section
Captureandh

File: index.html [hunk 2]
```diff
@@ -243,12 +256,17 @@ <h3 class="connectlink">LET´S CONNECT</h3>
             </ul>
           </div>
           <div class="link-project"></div>
-          <a class="linktonic" href="#">Get my resume</a>
+<div class="resume-link">
+          <a href="#">Get my resume</a>
```
Comment (Kossi-stack) [inline index.html:262]: - [ ] Kindly fix the resume button when is pressed, : hover

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @andym80,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️
 - [ ] Kindly capitalize the first of your commit message in the next milestone so it looks professional.
 - [ ] Make sure you follow the [Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) provided.
 
 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (omarsalem7) [reply index.html:80]: - [ ] As the previous reviewer mentioned, please fix your design to math Figma design

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |

File: index.html (hunk 2 above)
Comment (omarsalem7) [reply index.html:262]: - [ ] Kindly fix hover in button. This is code color for hover button: `#6070FF;`
- [ ] [OPTIONAL] Consider fixing the hover color for other buttons also.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @andym80, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by andym80** [comment]
Screenshot from 2022-06-11 11-39-01
This is my view

**Review by topeogunleye — APPROVED** [review]
### Hi @andym80, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**